host/linux/dynload.c

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include "../dynload.h"
+#include <dlfcn.h>
+
+void* oe_shared_library_load(const char* name)
+{
+    return dlopen(name, RTLD_LAZY | RTLD_LOCAL);
```

Comment:
`RTLD_LOCAL` changes the semantics of this slightly for the sgx_common_library functions since previously based on static linkage, all the symbols were global. I don't know if it matters or if this is preferred?

More generally, this still makes me nervous as the security guidance on the Linux side has been towards `BIND_NOW`, and Linux doesn't have the same kinds of controls for shared library load pathing by the app? I don't know of a better solution though.

Reply:
Addressed in PR #3155